wordpress-mobile/WordPress-Android
# Detekt - Resolve/Suppress All Baseline Warnings - Long methods warnings_ MediaPickerActivity

Parent: https://github.com/wordpress-mobile/WordPress-Android/issues/17010

This PR resolved/suppress all complexity related LongMethod warnings for the WordPress module (see docs [here](https://detekt.dev/docs/rules/complexity#longmethod)):

`1` x LongMethod (Resolve: https://github.com/wordpress-mobile/WordPress-Android/commit/e06b430a27042a7fd755353d71af5cae65e5b47c + https://github.com/wordpress-mobile/WordPress-Android/commit/6086e9f25029ff3d1bc798742559699fb1308c35)
To test:

- There is nothing much to test here.

- Verifying that all the CI checks are successful should be enough (especially the detekt check).

- However, if you really want to be thorough, you could smoke test the WordPress and/or Jetpack apps to verify that everything works as expected on every screen that relates to these changes.
    STEPS
- Go to the menu tab
- click on media option 
- click on the upload media button or the (+) button at the top right corner 
- select choose from device 
- select an image

## Regression Notes
1. Potential unintended areas of impact
     can't think of any 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
   See `To test` section above.

3. What automated tests I added (or what prevented me from doing so)
     N/A

PR submission checklist:
